src/views/pages/Swap/Swap.js

```diff
@@ -171,8 +171,9 @@ const Swap = () => {
   const swapInput2 = document.getElementById('swapInput2')
 
   const clearInputs = async () => {
-    swapInput1.value = '0'
-    swapInput2.value = '0'
+    swapInput1.value = ''
+    swapInput2.value = ''
+    swapInput1.focus()
```

Comment:
On clearInput; focus is switched to the 1st input (UX especially for mobile users)
Clears to placeholder instead of 0 now for the same reason (UX)